docker_master.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+docker-compose up mongo postgres &
```

Comment:
whats the differece to 'docker-compose up -d ...'?

Reply:
probably the same, changed